operate/wallet/wallet_recovery_manager.py

```diff
@@ -412,6 +435,17 @@ def _report_issue(msg: str) -> None:
             )
             for file in new_root.glob(f"{USER_JSON}*"):
                 shutil.copy2(file, root / file.name)
+            for file in new_keys_path.iterdir():
+                shutil.copy2(file, root / KEYS_DIR / file.name)
+
+            new_agent_keys = self.data.new_agent_keys
+            for service in self.service_manager.get_all_services()[0]:
+                service_config_id = service.service_config_id
+                service.agent_addresses = [
+                    new_agent_keys[service_config_id].get(addr, addr)
```

Comment:
Here in `.get(addr, addr)`, if the `addr` is not found in `new_agent_keys[service_config_id]` then we're taking that same address in the `new_agent_keys`? But that's an error, because if in this case something went wrong in the preparation step. Lmkyt.